lib/standalone_migrations/generator.rb

```diff
@@ -8,4 +9,14 @@ def self.migration(name, options="")
         :destination_root => Rails.root
     end
   end
+
+  class CacheMigrationGenerator < ActiveRecord::Generators::MigrationGenerator
```

Comment:
Is this class supposed to be part of the pr?

Reply:
yes, the migration templates in rails are hardcoded in rails 4 
https://www.rubydoc.info/docs/rails/4.1.7/ActiveRecord%2FGenerators%2FMigrationGenerator:create_migration_file

and with another configuration with rails 5
https://github.com/rails/rails/blob/7bc343b3a9ef7581e665a9f70f00d494f7a3b62c/activerecord/lib/rails/generators/active_record/migration.rb

the easiest workaround is to overwrite the method  to support both versions

Reply:
Gotcha, thanks for the explanation.